lib/ide/ast/impl/src/lib.rs

```diff
@@ -895,6 +900,11 @@ impl Ast {
     // TODO smart constructors for other cases
     //  as part of https://github.com/luna/enso/issues/338
 
+    pub fn number(number:i64) -> Ast {
+        let number = Number { base:None,int:number.to_string() };
```

Comment:
minor thing - if we put no spaces around `,` then there sohuld be no space after `{` and before `}` as well.